### DISCUSSION TITLE
Source code upgrade

### DISCUSSION THREAD

**Discussion by LuisVilaplana** [oqtane/oqtane.framework#3335] [General] [answered]
What is the official suggested procedure to update the Oqtane source code to a new version?

I am using Visual Studio 2022 and SQL Server. I did the initial installation of Oqtane by downloading the sources from Github (zip) and I have not modified anything apart from the connection strings.

Thanks

**Comment by sbwalker** [comment, marked as answer]
You should be able to download the latest source and unzip it over top of your existing code (make sure you preserve your appsettings.json). Recompile and run and any migrations will be executed automatically.

**Reply by LuisVilaplana** [reply]
Ok, I will, thanks
